recipes/pastml/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "pastml" %}
+{% set version = "1.9.38" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 43bf7d2a3a9b9b67da7c5881ecdeb2ee9cccd1585b8f65700e53761609945cc6
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pastml = pastml.acr:main
+    - geomap = pastml.visualisation.generate_geomap:main
+    - transition_counter = pastml.utilities.transition_counter:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
I was just reviewing the host requirements. I believe you only need to list python and pip here. And in run below, you don't need pip.

Reply:
Indeed, the package dependencies  were only needed for run and not host. Thank you for spotting it @andersgs ! I have now fixed them